storage/container_storage_interface/ephemeral-storage-csi-inline.adoc

```diff
@@ -10,10 +10,6 @@ Container Storage Interface (CSI) inline ephemeral volumes allow you to define a
 
 This feature is only available with supported Container Storage Interface (CSI) drivers.
 
-:FeatureName: CSI inline ephemeral volumes
```

Comment:
We need to document the security considerations / admission plugin
https://github.com/openshift/enhancements/blob/master/enhancements/storage/csi-inline-vol-security.md
@dobsonj

Reply:
Done.